### DISCUSSION TITLE
How to use token protected self hosted interactsh server with Nuclei

### DISCUSSION THREAD

**Discussion by fdellagr** [projectdiscovery/nuclei#2467] [Q&A] [answered]
Hi,

I am trying to run a command that calls my self hosted interactsh url but nuclei keeps erroring out saying the following 

`nuclei -u https://hostname -tags log4j -iserver custom.intershurl.fun -v`

`Could not register to (custom.intershurl.fun): invalid token provided for interactsh server, retrying with remaining
[ERR] Could not initialize interactsh client: could not create client: could not register to servers: invalid token provided for interactsh server`

Am I using the -server command incorrectly?

**Comment by ehsandeep** [comment, marked as answer]
@fdellagr as the error indicates, you are running token protected interactsh server, so you also need to pass the server token using `-itoken` flag to connect your server.

> invalid token provided for interactsh server

**Reply by fdellagr** [reply]
Thanks but now getting this error 

`Could not initialize interactsh client: could not create client: could not register to servers: could not make register request: POST (custom.intershurl.fun) giving up after 1 attempts: Post "(custom.intershurl.fun)": dial tcp ipaddress: i/o timeout`

**Reply by fdellagr** [reply]
@ehsandeep any idea why the above error is happening?

**Reply by ehsandeep** [reply]
@fdellagr to confirm on this further, we will required `custom.intershurl.fun` to see if it's an issue with interactsh server or nuclei or anything else, if you are fine with sharing that information, msg me at https://discord.gg/projectdiscovery (pdteam)

**Reply by fdellagr** [reply]
@ehsandeep the interactsh server is internally hosted so you wouldnt be able to reach it
